sycl/test-e2e/XPTI/basic_event_collection_linux.cpp

```diff
@@ -43,8 +43,6 @@
 // CHECK:      UR Call Begin : urKernelSetArgPointer
 // CHECK-NEXT: UR Call Begin : urKernelGetGroupInfo
 // CHECK-NEXT: UR Call Begin : urEnqueueKernelLaunch
-// CHECK-NEXT: UR Call Begin : urKernelRelease
```

Comment:
Kind of same question here, why these calls are removed. It seems that they should be just moved.

Reply:
Sure.